### PR TITLE
🇶🇦 Fix Errors from code ql scanning

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes
Fix partial issues from code ql scanning

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.
- [ ] Is this PR including examples changes? If yes, please remember to update [example documentation](https://github.com/microsoft/Olive/blob/main/docs/source/examples.md) in a follow-up PR.

### CODE REVIEW

**Comment by trajepl** [comment]
Check the CodeQL scanning results:
https://github.com/microsoft/Olive/security/code-scanning?query=is%3Aopen+pr%3A1081

File: olive/passes/pytorch/lora.py [hunk 1]
```diff
@@ -588,6 +588,7 @@ def train_and_save_new_model(
         :param output_model_path: The path to save the output model to.
         :return: The output model handler.
         """
+        allow_tf32 = None
```
Comment (guotuofeng) [inline olive/passes/pytorch/lora.py:591]: we can ignore the warning here since allow_tf32 is only defined for cuda.

Comment (trajepl) [reply olive/passes/pytorch/lora.py:591]: This kind of isssues are shown as `errors`.

Comment (guotuofeng) [reply olive/passes/pytorch/lora.py:591]: I mean we might need find some ways to ignore/disable this errors since we might not be able fix all errors.

**Review by guotuofeng — APPROVED** [review]

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**